nmpc_controller/src/nmpc_controller.cpp

```diff
@@ -92,8 +158,8 @@ NMPCController::NMPCController(int type) {
   app_->Options()->SetNumericValue("warm_start_slack_bound_push", 1e-6);
   app_->Options()->SetNumericValue("warm_start_mult_bound_push", 1e-6);
 
-  app_->Options()->SetNumericValue("max_wall_time", 3.0 * dt_);
-  app_->Options()->SetNumericValue("max_cpu_time", 3.0 * dt_);
+  app_->Options()->SetNumericValue("max_wall_time", 10.0 * dt_);
+  app_->Options()->SetNumericValue("max_cpu_time", 10.0 * dt_);
```

Comment:
If you'd like to apply it to hardware, we might want to set this to a smaller value

Reply:
Good call, updated.